packit_service/config.py

```diff
@@ -47,16 +48,25 @@ class Deployment(enum.Enum):
 
 
 class ServiceConfig(Config):
-    SCHEMA = SERVICE_CONFIG_SCHEMA
+    # SCHEMA = SERVICE_CONFIG_SCHEMA
+
+    def __init__(
+        self,
+        deployment: Deployment = Deployment.stg,
+        webhook_secret: str = "",
+        testing_farm_secret: str = "",
+        validate_webhooks: bool = True,
+        **kwargs,
+    ):
 
-    def __init__(self):
-        super().__init__()
+        super().__init__(**kwargs)
 
-        self.deployment: Deployment = Deployment.stg
+        self.deployment = deployment
 
-        self.webhook_secret: str = ""
-        self.testing_farm_secret: str = ""
-        self.validate_webhooks: bool = True
+        # duplicate - also in Config -> can be removed?
+        self.webhook_secretr = webhook_secret
```

Comment:
No idea why it's also in `Config` when we AFAIK [use it only in service](https://github.com/packit-service/packit-service/blob/master/packit_service/service/api/webhooks.py#L110)

BTW
```suggestion
        self.webhook_secret = webhook_secret
```